build/grunt.js

```diff
@@ -326,32 +318,39 @@ module.exports = function(grunt) {
       }
     },
     browserify: {
-      options: browserifyGruntOptions(),
       build: {
```

Comment:
Yeah, the difference between the two was that `dist` would  replace the vtt.js string path with a url to the cdn. Since we will require vtt.js by default when we build it isn't really needed to do them separately